cairo-sys-rs/src/lib.rs

```diff
@@ -89,16 +92,14 @@ debug_impl!(cairo_t);
 pub struct cairo_surface_t(c_void);
 debug_impl!(cairo_surface_t);
 
-#[repr(C)]
-#[derive(Copy,Clone,Debug)]
-pub struct cairo_pattern_t(u8);
-
-#[cfg(any(feature = "xcb", feature = "dox"))]
 #[repr(C)]
 pub struct cairo_device_t(c_void);
-#[cfg(any(feature = "xcb", feature = "dox"))]
 debug_impl!(cairo_device_t);
 
+#[repr(C)]
+#[derive(Copy,Clone,Debug)]
+pub struct cairo_pattern_t(u8);
```

Comment:
Elsewhere we use a tuple struct around `c_void` for this. That's probably a better idea, also to keep it consistent (please check if other things have the same problem).